volplugin/utils.go

```diff
@@ -164,3 +189,26 @@ func splitPath(name string) (string, string, error) {
 func joinPath(tenant, name string) string {
 	return strings.Join([]string{tenant, name}, ".")
 }
+
+func addStopChan(name string) chan struct{} {
+	mountMutex.Lock()
+	stopChan := make(chan struct{}, 1)
+
+	if sc, ok := mountStopChans[name]; ok {
```

Comment:
shouldn't it be an error if a channel already exists with that name? I might being a bit conservative here :)